### PR TITLE
Jokes

### PR DESCRIPTION
**Pull request by sveinbjornt**
* Renamed frivolous.py to special.py
* Added joke responses and answers to queries about capabilities ("Hvaða spurningar skilurðu?")
* Minor improvements in geography module
* Fixed entity query tests. The articles and persons database tables are now populated with dummy data from CSV files stored in tests/test_files

### CODE REVIEW

File: routes/main.py [hunk 1]
```diff
@@ -290,7 +290,10 @@ def parsefail():
         sfails = []
 
         for a in q.all():
-            tokens = json.loads(a.tokens)
+            try:
+                tokens = json.loads(a.tokens)
+            except:
+                continue
```
Comment (vthorsteinsson) [inline routes/main.py]: `tokens = []`

File: tests/test_files/testdb_articles.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+https://greynir.is/,2f1963a4-34fe-11e9-a615-3200174ea5c0,3,Grein,Höfundur,2019-02-20 07:00:00,0.8,2019-02-20 10:56:51.362325,2019-02-20 11:02:24.189203,2019-02-20 16:25:32.738527,2019-04-17 15:23:21.808573,scrapers.default,VisirScraper,1.0,2019-02-14 11:41:47/1.0/1.0,9,8,1.61813453744468,"","","",""
```
Comment (vthorsteinsson) [inline tests/test_files/testdb_articles.csv]: Add missing LF at end of line

File: tests/test_files/testdb_persons.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+1,https://greynir.is/,Björn Þorsteinsson,prófessor í heimspeki,prófessor í heimspeki,kk,1,2019-02-11 13:44:41.635725
+2,https://greynir.is/,Viðar Þorsteinsson,framkvæmdastjóri Eflingar,framkvæmdastjóri fflingar,kk,1,2019-02-11 13:44:41.992479
```
Comment (vthorsteinsson) [inline tests/test_files/testdb_persons.csv]: `fflingar` -> `Eflingar`

File: routes/main.py (hunk 1 above)
Comment (sveinbjornt) [reply routes/main.py]: Why?

Comment (vthorsteinsson) [reply routes/main.py]: Sorry, didn't see the context. This is fine as-is.

**Pull request merged**